setup.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import re
+import os
+from setuptools import setup, find_packages
+
+
+with open('secure_redis/__init__.py', 'r') as init_file:
+    version = re.search('^__version__ = [\'"]([^\'"]+)[\'"]', init_file.read(), re.MULTILINE).group(1)
+
+# allow setup.py to be run from any path
+os.chdir(os.path.normpath(os.path.join(os.path.abspath(__file__), os.pardir)))
+
+setup(
+    name='django-redis-secure',
+    version=version,
+    packages=find_packages(),
+    # license='MIT License',
+    # include_package_data=True,
+    description=(
+        'Django redis client and serializer to ensure data encreption and providing more data security.'
```

Comment:
also, encryption is spelled incorrectly. Perhaps change to:

"Django caching plugin for django-redis that adds a Serializer class and configuration to support transparent, symmetrical encryption of cached values using the python cryptography library"